.github/assets/js/command-palette.js

```
@@ -0,0 +1,315 @@
+  
+  // Clear results
+  resultsContainer.innerHTML = '';
+  
+  // Get commands
+  const commands = window.commandData || [];
+  
+  // Filter commands based on query
+  const filteredCommands = query ? 
+    commands.filter(cmd => 
+      cmd.title.toLowerCase().includes(query.toLowerCase()) ||
+      (cmd.section && cmd.section.toLowerCase().includes(query.toLowerCase()))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)